netmap/netmap_contract.go

```diff
@@ -685,9 +703,9 @@ func setConfig(ctx storage.Context, key, val any) {
 }
 
 func cleanup(ctx storage.Context, epoch int) {
-	balanceContractAddr := storage.Get(ctx, balanceContractKey).(interop.Hash160)
-	contract.Call(balanceContractAddr, cleanupEpochMethod, contract.All, epoch)
-
-	containerContractAddr := storage.Get(ctx, containerContractKey).(interop.Hash160)
-	contract.Call(containerContractAddr, cleanupEpochMethod, contract.All, epoch)
+	it := storage.Find(ctx, newEpochRecipientsKey, storage.RemovePrefix|storage.KeysOnly)
```

Comment:
Does order of invocations matter here?

Reply:
you mean Container -> Balance or Balance -> Container? no

Reply:
you mean Container -> Balance or Balance -> Container? no

Reply:
Can it become important in the future? We can register them with prefix_index_hash key, just in case.

Reply:
> Can it become important in the future?

anything can happen

> We can register them with prefix_index_hash

what exactly is `index` in your suggestion? registration number?

Reply:
Yeah, a simple byte will likely work, can't imagine 255+ contracts subscribing to this. At least there would be some predictability in the order, otherwise it's simply hash-dependent and will vary from network to network. Who knows how it can affect us in the future.

Reply:
done